benchmark/policy/policy-startup.js

```diff
@@ -0,0 +1,52 @@
+// Tests the impact on eager operations required for policies affecting
+// general startup,  does not test lazy operations
+'use strict';
+const common = require('../common.js');
+
+const configs = {
+  n: [1024]
+};
+
+const options = {
+  flags: ['--expose-internals']
+};
+
+const bench = common.createBenchmark(main, configs, options);
+
+function main(conf) {
+  const hash = (str, algo) => {
+    const hash = require('crypto').createHash(algo);
+    hash.update(str);
+    return hash.digest('base64');
```

Comment:
FWIW this can be simplified to `return hash.update(str).digest('base64');`

Reply:
updated